backend/protocol_rpc/endpoints.py

```diff
@@ -52,6 +52,22 @@
 from flask import request
 from flask_jsonrpc.exceptions import JSONRPCError
 import base64
+import os
+
+
+####### WRAPPER TO BLOCK ENDPOINTS FOR HOSTED ENVIRONMENT #######
+def check_vite_hosted(func):
```

Comment:
@kstroobants this is not a good name for this wrapper. It should be something similar to `check_forbidden_method_in_hosted_studio`